solace-spring-cloud-starters/solace-spring-cloud-stream-starter/README.adoc

```diff
@@ -278,6 +278,11 @@ The following properties are available for Solace producers only and must be pre
 
 See link:../../solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/properties/SolaceCommonProperties.java[SolaceCommonProperties] and link:../../solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/properties/SolaceProducerProperties.java[SolaceProducerProperties] for the most updated list.
 
+headerExclusions::
+To exclude headers on sending that are not serializable, or you want to shrink data volume usage, you can exclude headers by their name.
```

Comment:
Change this description to:
`The list of headers to exclude from the published message. This will not filter out any Solace message headers.`

Reply:
changed